core/lib/spree/core/importer/order.rb

```diff
@@ -98,19 +100,31 @@ def self.create_shipments_from_params(shipments_hash, order)
 
         def self.create_line_items_from_params(line_items_hash, order)
           return {} unless line_items_hash
-          line_items_hash.each_key do |key|
-            extra_params = line_items_hash[key].except(:variant_id, :quantity, :sku)
-            line_item = ensure_variant_id_from_params(line_items_hash[key])
-            line_item = order.contents.add(Spree::Variant.find(line_item[:variant_id]), line_item[:quantity])
-            # Raise any errors with saving to prevent import succeeding with line items failing silently.
-            if extra_params.present?
-              line_item.update!(extra_params)
-            else
-              line_item.save!
+
+          case line_items_hash
+          when Hash
+            line_items_hash.each_key do |key|
```

Comment:
What's the reason you're not using `each_value` instead? Or better yet, converting `line_items_hash` so that it's always an array, and _then_ iterating on the converted array to call `create_line_item`?

Reply:
Thanks for notice that @aldesantis 

That comes from the original implementation, but you are right could be better use `each_value` instead.

Reply:
Sorry, I didn't realize it came from the original implementation. I think it makes sense not to change it in this PR, if that's the case!